packages/usa-in-page-navigation/src/test/template.html

```diff
@@ -0,0 +1,23 @@
+<div class="usa-in-page-nav-container">
+  <main id="main-content" class="main-content">
+    <h1>In page navigation test heading</h1>
+
+    <h2><a id="section_0" class="usa-anchor-tag"></a>Section 1</h2>
+    <p></p>
+    <h3><a id="section_1" class="usa-anchor-tag"></a>Section 1.1</h3>
+    <p>Section 1.1 content</p>
+    <h3><a id="section_2" class="usa-anchor-tag"></a>Section 1.2</h3>
+    <p>Section 1.2. content</p>
+  </main>
+
+  <aside class="usa-in-page-nav">
+    <nav role="navigation" arial-label="In-page navigation">
```

Comment:
Little typo. Additionally, `<nav>` has an implicit role of "navigation", so I believe there is no need to make it explicit. (Reference: [Mozilla](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav))

```suggestion
    <nav aria-label="In-page navigation">
```

Reply:
Just pushed the fix, thank you!